css/style.css

```diff
@@ -0,0 +1,244 @@
+:root {
+    --main-color: #2874f0;
+}
+
+html,
+body {
+    padding: 0%;
+    margin: 0%;
+}
+
+.bg-system {
+    background-color: var(--main-color) !important;
+}
+
+.navbar {
+    background-color: var(--main-color);
```

Comment:
what is the use of var() here ?

Reply:
it's used for the default color every website has its default theme color so that's the reason I have used var here.